source/fe/fe.cc

```diff
@@ -962,7 +962,18 @@ FiniteElement<dim, spacedim>::hp_quad_dof_identities(
 template <int dim, int spacedim>
 FiniteElementDomination::Domination
 FiniteElement<dim, spacedim>::compare_for_face_domination(
-  const FiniteElement<dim, spacedim> &) const
+  const FiniteElement<dim, spacedim> &fe_other) const
+{
+  return this->compare_for_domination(fe_other, 1);
```

Comment:
if you search for this line and find that any derived class implements the function this way, then you should just be able to remove the function you find because it overloads the base class's implementation with the same implementation.